test/integration/executor/executor_fixture.hpp

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef TEST_INTEGRATION_EXECUTOR_FIXTURE_HPP
+#define TEST_INTEGRATION_EXECUTOR_FIXTURE_HPP
+
+#include "framework/executor_itf/executor_itf.hpp"
+
+#include <gtest/gtest.h>
+#include "common/result.hpp"
+#include "cryptography/crypto_provider/crypto_defaults.hpp"
+#include "framework/common_constants.hpp"
+#include "framework/result_gtest_checkers.hpp"
+#include "integration/executor/executor_fixture_param.hpp"
+#include "interfaces/query_responses/query_response.hpp"
+#include "module/shared_model/mock_objects_factories/mock_command_factory.hpp"
+#include "module/shared_model/mock_objects_factories/mock_query_factory.hpp"
+
+namespace executor_testing {
+
+  namespace error_codes {
+    using shared_model::interface::ErrorQueryResponse;
+
+    // TODO [IR-1816] Akvinikym 06.12.18: remove these constants after
+    // introducing a uniform way to use them in code
+    static constexpr ErrorQueryResponse::ErrorCodeType kNoStatefulError = 0;
```

Comment:
Maybe introduce an Enum?

Reply:
decided that a desireable solution falls under the present todo